### ISSUE TITLE
The difference between the generated image and the training image is too large

### ISSUE THREAD

**Issue by tdf1995** [mit-han-lab/data-efficient-gans#99]
train_set:
![image](https://user-images.githubusercontent.com/39071957/174749833-f02645a7-46ba-4e2d-82c2-aeda99dfeec9.png)
generated images:
![image](https://user-images.githubusercontent.com/39071957/174749451-b533a298-7fa7-4f12-8e42-038acf91e339.png)

@click.option('--outdir',default='/work/ai_lab/miner/data-efficient-gans/output', help='Where to save the results', required=True, metavar='DIR')
@click.option('--gpus',default=2, help='Number of GPUs to use [default: 1]', type=int, metavar='INT')
@click.option('--snap',default=10, help='Snapshot interval [default: 50 ticks]', type=int, metavar='INT')
@click.option('--metrics', help='Comma-separated list or "none" [default: fid50k_full]', type=CommaSeparatedList())
@click.option('--seed', help='Random seed [default: 0]', type=int, metavar='INT')
@click.option('-n', '--dry-run', help='Print training options and exit', is_flag=True)

# Dataset.
@click.option('--data',default=r'/work/ai_lab/miner/data-efficient-gans/bicycle', help='Training data (directory or zip)', metavar='PATH', required=True)
@click.option('--cond', help='Train conditional model based on dataset labels [default: false]', type=bool, metavar='BOOL')
@click.option('--subset', help='Train with only N images [default: all]', type=int, metavar='INT')
@click.option('--mirror',default=True, help='Enable dataset x-flips [default: false]', type=bool, metavar='BOOL')

# Base config.
@click.option('--cfg', help='Base config [default: low_shot]', type=click.Choice(['low_shot', 'auto', 'stylegan2', 'paper256', 'paper512', 'paper1024', 'cifar']))
@click.option('--gamma', help='Override R1 gamma', type=float)
@click.option('--kimg', help='Override training duration', type=int, metavar='INT')
@click.option('--batch',default=8, help='Override batch size', type=int, metavar='INT')

# Discriminator augmentation.
@click.option('--DiffAugment', help='Comma-separated list of DiffAugment policy [default: color,translation,cutout]', type=str)
@click.option('--aug', help='Augmentation mode [default: ada]', type=click.Choice(['noaug', 'ada', 'fixed']))
@click.option('--p', help='Augmentation probability for --aug=fixed', type=float)
@click.option('--target', help='ADA target value for --aug=ada', type=float)
@click.option('--augpipe', help='Augmentation pipeline [default: bgc]', type=click.Choice(['blit', 'geom', 'color', 'filter', 'noise', 'cutout', 'bg', 'bgc', 'bgcf', 'bgcfn', 'bgcfnc']))

# Transfer learning.
@click.option('--resume', help='Resume training [default: noresume]', metavar='PKL')
@click.option('--freezed', help='Freeze-D [default: 0 layers]', type=int, metavar='INT')

# Performance options.
@click.option('--fp32', help='Disable mixed-precision training', type=bool, metavar='BOOL')
@click.option('--nhwc', help='Use NHWC memory format with FP16', type=bool, metavar='BOOL')
@click.option('--nobench', help='Disable cuDNN benchmarking', type=bool, metavar='BOOL')
@click.option('--allow-tf32', help='Allow PyTorch to use TF32 internally', type=bool, metavar='BOOL')
@click.option('--workers',default=4, help='Override number of DataLoader workers', type=int, metavar='INT')

could you give me some advice to make the results better?Thanks
